tests/incomplete.pyi

```diff
@@ -0,0 +1,20 @@
+from _typeshed import Incomplete
+
+att: Incomplete
+
+def ok(x: Incomplete | None) -> list[Incomplete]: ...
+def err1(
+    x: Incomplete,  # Y065 Don't use bare "Incomplete", leave unannotated
+) -> None: ...
+def err2() -> Incomplete: ...  # Y065 Don't use bare "Incomplete", leave unannotated
+
+class Foo:
+    att: Incomplete
+    def ok(self, x: Incomplete | None) -> list[Incomplete]: ...
+    def err1(
+        self,
+        x: Incomplete,  # Y065 Don't use bare "Incomplete", leave unannotated
+    ) -> None: ...
+    def err2(
+        self,
+    ) -> Incomplete: ...  # Y065 Don't use bare "Incomplete", leave unannotated
```

Comment:
It would be nice if we could tailor the error message slightly more, e.g.:

```suggestion
    x: Incomplete,  # Y065 Leave parameter "x" unannotated rather than using "Incomplete"
) -> None: ...
def err2() -> Incomplete: ...  # Y065 Omit the return annotation rather than using "Incomplete"

class Foo:
    att: Incomplete
    def ok(self, x: Incomplete | None) -> list[Incomplete]: ...
    def err1(
        self,
        x: Incomplete,  # Y065 Leave parameter "x" unannotated rather than using "Incomplete"
    ) -> None: ...
    def err2(
        self,
    ) -> Incomplete: ...  # Y065 Omit the return annotation rather than using "Incomplete"
```